### PR TITLE
Refactor `%browse` to use the same code as `%head`

### LINKED ISSUE #215: %browse sends previous dataset even when no data in memory
To reproduce:
```stata
sysuse auto
%browse
clear
%browse
```

Note that `%head` doesn't have the same issue.

### PR DESCRIPTION
**Pull request by kylebarron**
Supersedes https://github.com/kylebarron/stata_kernel/pull/89.

Fixes #215

### CODE REVIEW

**Comment by kylebarron** [comment]
This is way easier than having separate code for `%browse` and `%head` when they do the same thing.
@mcaceresb

File: stata_kernel/ado/_StataKernelHead.ado [hunk 1]
```diff
@@ -15,7 +15,7 @@ program _StataKernelHead
         if ( _rc ) error 198
         local n = `n1'
     }
-    else local n = 10
+    else local n = `N_default'
```
Comment (mcaceresb) [inline stata_kernel/ado/_StataKernelHead.ado]: I don't think this works bc uppercase is a shortcut. Should be `n_default`

Comment (kylebarron) [reply stata_kernel/ado/_StataKernelHead.ado]: oh lol yeah

**Comment by kylebarron** [comment]
Also, this is hitting the same issue as I mentioned in #89. Using `sysuse auto`, `%browse` says
```
error: observation numbers out of range
```
I think this is because I'm asking for N from 1/200, and there aren't 200 obs

**Comment by mcaceresb** [comment]
`local n = min(_N, 200)` ?

**Comment by kylebarron** [comment]
Seems to work good for me!

**Pull request merged**